src/NServiceBus.Persistence.CosmosDB/SynchronizedStorage/ICosmosStorageSession.cs

```diff
@@ -5,7 +5,7 @@
     /// <summary>
     ///
```

Comment:
I'm assuming this will be a separate PR with XML doco updates.

Reply:
See https://github.com/Particular/NServiceBus.Persistence.CosmosDB/pull/67